### PR TITLE
Theory questions

### PR DESCRIPTION
**Pull request by Xiaoxuan3x**

### CODE REVIEW

File: S1:Theory_Question.md
Comment (abigailsleek) [inline S1:Theory_Question.md:1]: 1. Answer is correct.
2. Answer is partially correct; it is a type of arithmetic exception called ZeroDivisionError
3. Answer is correct
4. Answer is correct
5. No response was given
6. Answers given were not explained
7. Explanation is correct
8. Answer is correct.
9. Good requirements, considerations, challenges and components highlighted.

**Review by abigailsleek — COMMENTED** [review]
Good work done!

**Comment by abigailsleek** [comment]
Kindly merge this request to the main as I do not have write access to do so.

**Comment by Xiaoxuan3x** [comment]
> Kindly merge this request to the main as I do not have write access to do so.
I've merged it to main. thanks!

**Pull request merged**